src/components/modal/tests/ModalHeaderConnected.spec.tsx

```diff
@@ -31,6 +31,7 @@ describe('ModalHeader', () => {
                     <ModalHeaderConnected
                         id={id}
                         title={title}
+                        lastOpened={true}

```

Comment:
if this is a boolean prop in react, only putting `lastOpened` means the same as `lastOpened={true}`